cmd/kubelet/app/server.go

```diff
@@ -173,9 +173,9 @@ func NewKubeletServer() *KubeletServer {
 		FileCheckFrequency:          20 * time.Second,
 		HealthzBindAddress:          net.ParseIP("127.0.0.1"),
 		HealthzPort:                 10248,
-		HostNetworkSources:          kubelet.FileSource,
-		HostPIDSources:              kubelet.FileSource,
-		HostIPCSources:              kubelet.FileSource,
+		HostNetworkSources:          kubelet.AllSource,
```

Comment:
Please update the flag documentation also:
https://github.com/kubernetes/kubernetes/blob/7f9df8c062a693fb05923b1407d5995724f96b44/cmd/kubelet/app/server.go#L231